### DISCUSSION TITLE
Which country’s flag bears the motto "Ordem e Progresso"?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#833] [Galaxy Trivia] [answered]
**Bonus:** How many ⭐⭐ does it have and what is the reason for this number?

**Comment by peterbaccaro** [comment, marked as answer]
The country whose flag bears the motto "Ordem e Progresso" is **Brazil**. The phrase, which translates to "Order and Progress" in English, is inscribed on the flag of Brazil.

The flag of Brazil features **27 white stars**. These stars represent the 26 states of Brazil and the Federal District. The number of stars has remained constant since the flag was first adopted in 1889.

The meaning of the stars on the Brazilian flag is to **represent the night sky over Rio de Janeiro on November 15, 1889, the date when the Brazilian Republic was proclaimed**. The stars are arranged in the pattern of the night sky as seen from Rio de Janeiro on that particular date, with each star representing a specific constellation. The arrangement of the stars is a unique representation of the celestial sphere over Brazil at that specific moment in history.

**Comment by SANKARAMDAS** [comment]
Brazil.
The field is green, with a yellow diamond bearing a blue celestial globe with the constellations of the Southern Hemisphere and the motto: 'ORDEM E PROGRESSO' (order and progress). The national colours are those adopted in 1822 when Brazil declared its independence from Portugal under Emperor Pedro I.

**Comment by anjutares** [comment]
The country whose flag bears the motto "Ordem e Progresso" is Brazil. The flag of Brazil features a green field with a large yellow diamond in the center, within which there is a blue circle with 27 white five-pointed stars. The stars represent the 26 states of Brazil and the Federal District, with each star symbolizing a specific state. The arrangement of the stars corresponds to the constellation of the night sky over Rio de Janeiro on November 15, 1889, which was the date when the Brazilian republic was proclaimed.
